src/strategies/ocean-dao-brightid/examples.json

```diff
@@ -4,86 +4,102 @@
     "strategy": {
       "name": "ocean-dao-brightid",
       "params": {
-        "brightIdMultiplier": 5,
+        "brightIdMultiplier": 4,
         "notVerifiedMultiplier": 1,
+        "brightIdNetwork": "1",
         "registry": "0xc37F8341Ac6e4a94538302bCd4d49Cf0852D30C0",
         "symbol": "OCEAN",
-        "strategies": [
-          {
-            "name": "erc20-balance-of",
-            "params": {
-              "symbol": "OCEAN",
-              "address": "0x967da4048cD07aB37855c090aAF366e4ce1b9F48",
-              "decimals": 18
-            }
-          },
-          {
-            "name": "ocean-marketplace",
-            "params": {
-              "symbol": "OCEAN",
-              "address": "0x967da4048cD07aB37855c090aAF366e4ce1b9F48",
-              "decimals": 18
-            }
-          },
-          {
-            "name": "sushiswap",
-            "params": {
-              "symbol": "OCEAN",
-              "address": "0x967da4048cD07aB37855c090aAF366e4ce1b9F48",
-              "decimals": 18
-            }
-          },
-          {
-            "name": "uniswap",
-            "params": {
-              "symbol": "OCEAN",
-              "address": "0x967da4048cD07aB37855c090aAF366e4ce1b9F48",
-              "decimals": 18
+        "graphs": {
+          "137": "https://api.thegraph.com/subgraphs/name/sameepsi/maticblocks"
+        },
+        "chains": {
```

Comment:
I think it would be better to keep it same as before (strategies with limits) 

If users want to get voting power from other networks, they can pass the `multichain` strategy into this strategy, right?

Reply:
@trizin You had a look at my comment?

Reply:
@bonustrack has told me that I can remove the limits. I was going to use the `multichain` strategy, but we only have one BrightId registry contract deployed and we'd like to only use that contract thus we are unable to use the `multichain` strategy directly. I've solved that problem using some code from the `multichain` strategy.

Reply:
Sorry for the confusion, I mean for this option `chains` , we can make it `strategies` like before right? I don't understand the need to have chains. users could pass a multichain strategy into this strategy and make it work for all chains right?


Also to get block with timestamp we now have a new API from snapshot, can you please make it the same as here https://github.com/snapshot-labs/snapshot-strategies/blob/7b60259ff6bb0eb79e8b2ec9b4cb5b29bb41d962/src/strategies/multichain/index.ts#L8-L27

Sorry for not mentioning it earlier

Reply:
Thanks, the new API works great! I updated the code.